person.rb

```diff
@@ -5,14 +5,15 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
```

Comment:
- [ ] I think it would be better if rentals are an `attr_accessor` so then it can be read and written 🙃